network/src/connectivity_manager/mod.rs

```diff
@@ -209,7 +209,10 @@ where
             eligible
         );
 
-        info!("{} connectivity manager init", network_context);
+        info!(
+            NetworkSchema::new(&network_context),
+            "{} Initialized connectivity manager", network_context
```

Comment:
You probably don't need to also include the network context in the formatting message, unless it adds value to the message itself, since the context will be included as structured data.

Reply:
I had this before you had added the structured data.  It can be removed.  It's mostly for readability of logs that would show up like:

```
[peer_id, role, network_id] Init connectiv manager
[peer_id, role, network_id] DId A
[peer_id, role, network_id] did B
[peer_id, role, network_id] did C
```

Reply:
Ah gotcha, I can see that being useful either way so I'll leave that up to your discretion